packages/foundry/contracts/YourContract.sol

```diff
@@ -0,0 +1,84 @@
+//SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0 <0.9.0;
+
+// Useful for debugging. Remove when deploying to a live network.
+import "forge-std/console.sol";
+
+// Use openzeppelin to inherit battle-tested implementations (ERC20, ERC721, etc)
+// import "@openzeppelin/contracts/access/Ownable.sol";
+
+/**
+ * A smart contract that allows changing a state variable of the contract and tracking the changes
+ * It also allows the owner to withdraw the Ether in the contract
```

Comment:
from the contract?

Reply:
> I don't know for sure, I'm not a native English speaker 😄 . Just noted that for me it looked a bit strange.

Lol same for me, and even I fell in trap on reading it first time! I think its better to be a bit verbose, just update it at https://github.com/scaffold-eth/scaffold-eth-2/pull/867/commits/52882e74a0b21f6e696e64f8c4083e9bf44d0f21